src/components/CytoscapePlayground.tsx

```diff
@@ -67,6 +69,69 @@ const editCursor =
 const trashCursor =
   "https://raw.githubusercontent.com/dvargas92495/roamjs-discourse-graph/main/src/cursors/trash.png";
 
+type ExportDialogProps = {
```

Comment:
This is a copy paste of the old `ExportDialog` component. will be nuked once we nuke Cytoscape since this was its last usage